mule-packager/src/main/java/org/mule/tools/api/packager/builder/PackageBuilder.java

```diff
@@ -10,26 +10,35 @@
 
 package org.mule.tools.api.packager.builder;
 
+import java.io.IOException;
+import java.nio.file.Path;
 
 import org.codehaus.plexus.archiver.ArchiverException;
 
-import java.io.File;
-import java.io.IOException;
-
 /**
  * Builder for packages.
  */
 public interface PackageBuilder {
 
   /**
-   * Cretes the package.
-   * @param destinationFile file to be created with the content of the package.
-   * @param originFolder location containing the resources that are going to be shipped in the package. It is expected that the
-   *        folder structure in this location is going to have the same structure of the contents of the generated package.
-   * @return
+   * Creates a package and it leaves it in the destinationPath. It does so based on a folder naming convention, it will look for
```

Comment:
"and it leaves it in " -> "and leaves that in" or "and copies that to" or "and writes that to"

Reply:
done